nvflare/fuel/hci/client/fl_admin_api.py

```diff
@@ -334,7 +334,7 @@ def delete_run(self, job_id: str) -> FLAdminAPIResponse:
         if not isinstance(job_id, str):
             raise APISyntaxError("job_id must be str but got {}.".format(type(job_id)))
         success, reply_data_full_response, reply = self._get_processed_cmd_reply_data(
-            AdminCommandNames.DELETE_RUN + " " + str(job_id)
+            AdminCommandNames.DELETE_WORKSPACE + " " + str(job_id)
         )
```

Comment:
maybe instead of `delete_run`, we should have `delete_job` method in this FLAdminAPI?

@nvkevlu what do you think?

But that change can come in different PR.

Reply:
yes, definitely, I will have that in my PR.